buy_together_app/migrations/0002_test_data.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('buy_together_app', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from buy_together_app.models import Product
```

Comment:
Any specific reason for having the import line here? It's generally a good practice to have all imports at the top of the module.

Reply:
We want the models to be importned in the inner function and not all the code,but i can fix it if you want

Reply:
Yes. Having it at the top is the proper way.

Reply:
Done